### PR TITLE
round out FAQs

### PR DESCRIPTION
**Pull request by mishmosh**
- Add more complete descriptions
- Add contact info
- Add how to participate

### CODE REVIEW

**Comment by mishmosh** [comment]
@b5 one more batch

File: devent.toml [hunk 1]
```diff
@@ -11,8 +11,7 @@ dateEnd = "2022-07-17"
 tagline="""
 _þing (n). thing, assembly, meeting, council. parliament._
 
-A collaborative series of workshops for IPFS Implementors.
-Hosted at the Harpa in Reykjavík, Iceland.
+Set in the inspiring and otherworldly landscape of Reykjavík, Iceland, IPFS þing is a week-long gathering for the IPFS implementors community. There will be talks, workshops, discussion circles, coworking and hacking time, and more — all focused on advancing IPFS implementations.
```
Comment (b5) [inline devent.toml:14]: daaaaaaaang nice wordsmithing

File: devent.toml [hunk 2]
```diff
@@ -69,17 +67,29 @@ teams = [
...
+  There is no charge to participate, and all meals during the event are provided. Participants (or their employers) are responsible for flights and accommodations. Several hotel blocks are being reserved and will be added here when confirmed; you are also free to book your own accommodations. 
+"""
+
+"Can I request travel or childcare assistance?" = """
+  Please email [ipfs-thing-2022@ipfs.io](mailto:ipfs-thing-2022@ipfs.io).
 """
 
 "How do I contact the organizers?" = """
-  You can reach out to us at _____@_____.
+  You can reach out to us at [ipfs-thing-2022@ipfs.io](mailto:ipfs-thing-2022@ipfs.io).
```
Comment (b5) [inline devent.toml:92]: is this live?

**Review by b5 — APPROVED** [review]
looks great. Thanks!

**Pull request merged**